### DISCUSSION TITLE
Temporarily disable an SSH key

### DISCUSSION THREAD

**Discussion by Suprabat** [community/community#22393] [New to GitHub]
<p>I have an office computer, accessible by other team members. I need to create an SSH key in my GitHub account in order to clone repositories into the office computer, but I do not want other members to access my Git account using that SSH key. Is it possible to create an SSH key and add it into my Git account, and then clone the repository I need, and then disable the key (not remove, just disable) on my Git account. Whenever I need to access the repo (pull, push etc.) in the office computer, I can temporarily enable the key in my account, and then disable it after using. Is this doable? What is the advice on this topic? Thanks in advance!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I don’t think there’s a way to temporarily disable an SSH key on your account. The best option that comes to mind would be to use an <a href="https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key-for-a-hardware-security-key">SSH key stored inside a hardware security token</a>, if that’s an option in your environment. That way the private key is never stored on the shared computer, and if you remove the hardware security key it’s definitely no longer available to any colleagues who may be logged in.</p>
